.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         tpu-type: ["v3-8", "v4-8", "v5e-4"]
     name: "TPU test (jaxlib=${{ matrix.jaxlib-version }}, ${{ matrix.tpu-type }})"
     env:
-      LIBTPU_VERSION_DATE: 20230908
+      LIBTPU_OLDEST_VERSION_DATE: 20230908
```

Comment:
I tested and found that 20230912 is the oldest version that has PJRT API version >= 0.29.

Reply:
Thanks, I'll update the date in this PR too (will be 6 weeks on Tuesday, close enough)